app/src/main/assets/sample-push-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pushServerURL": "pushServerURL (e.g http(s)//host:port/context)",
+  "android": {
+    "senderID": "senderID (e.g Google Project ID only for android)",
+    "variantID": "variantID (e.g. 1234456-234320)",
+    "variantSecret": "variantSecret (e.g. 1234456-234320)"
+  }
+}
```

Comment:
what's the purpose of this file? Is it used as an example? If so we probably should update the README file to describe how users should add the config file for push.

Reply:
Yes it is to be used as an example. I will be put into the README, that's the next thing that we are working on.